### ISSUE TITLE
Add Image Captioning examples in model zoo 

### ISSUE THREAD

**Issue by dexter1691** [Cloud-CV/Fabrik#119]
Image Captioning is another popular task for which we should have an example in Model Zoo

**Comment by Ram81** [comment]
@dexter1691 #120 contains a model for captioning MS-CoCo datset images to be added in ModelZoo :smile:

**Comment by dexter1691** [comment]
@Ram81 : This is just a language model, right? Can you find examples where both the CNN & Language Model are linked together?

**Comment by Ram81** [comment]
@dexter1691 Sure I can get one :smile:

**Comment by shahsaumya** [comment]
Can I work on this issue? @dexter1691

**Comment by Sayan98** [comment]
Hi, if this issue is unresolved, I'd love to take this up!

**Comment by RishabhJain2018** [comment]
@shahsaumya @Sayan98 This issue is still open. Please let us know if you are interested in working on it.

**Comment by thatbrguy** [comment]
@RishabhJain2018 Is image captioning the only model contribution you're accepting?

**Comment by deshraj** [comment]
@thatbrguy: no, please feel free to open an issue for a new model that you are planning to add. We can discuss more about it there. Does that sound good?

**Comment by Sayan98** [comment]
@RishabhJain2018 Okay, I'm taking this up.

**Comment by llucifer97** [comment]
Hi @dexter1691  if this issue is still open , I will take up this issuse
